packages/api-explorer/src/components/Markdown/common.tsx

```diff
@@ -43,14 +43,11 @@ MDCode.defaultProps = {
   pr: 'xxsmall',
 }
 
-// Temp fix for release build failure
-export const MDCodeBlock = styled(CodeBlock)`
+export const MDCodeBlock = styled(CodeBlock).attrs(
+  ({ fontSize = 'small', mb = 'large' }) => ({ fontSize, mb })
```

Comment:
Is this a new recommended pattern?

Reply:
Definitely within the @looker/components code-base it is :)

See: https://dev.to/layershifter/how-to-kill-tree-shaking-in-webpack-with-static-properties-p72

Reply:
Thanks for the link!

Reply:
I guess, because @lukelooker recommended it!

Reply:
I'm pretty sure that answer is almost as good as "I saw it on the internet so it _must_ be true" 😄